app/models/drink.rb

```diff
@@ -1,6 +1,7 @@
 class Drink < ActiveRecord::Base
   belongs_to :user
   has_many :drink_liquors
+  has_many :reviews
```

Comment:
You may want to add dependent: :destroy here so when a drink is deleted, its associated reviews are also deleted